docker-compose.yml

```diff
@@ -0,0 +1,50 @@
+version: '2'
+services:
+  BANZAIpipelineE2EData:
+    image: docker.lco.global/banzai-e2e-data:1.0.1
+    entrypoint:
+    - /bin/true
+    labels:
+      io.rancher.container.pull_image: always
+      io.rancher.container.start_once: 'true'
+    volumes:
+    - banzaie2e:/archive
+    logging:
+      options:
+        max-size: '100m'
+        max-file: '3'
+  AstrometryCatalogs:
+    image: docker.lco.global/astrometry.net-catalogs:1.0.0
+    entrypoint:
+    - /bin/true
+    labels:
+      io.rancher.container.pull_image: always
+      io.rancher.container.start_once: 'true'
+    volumes:
+    - astrometry:/opt/astrometry.net/data
+    logging:
+      options:
+        max-size: '100m'
+        max-file: '3'
+  BANZAITest:
+    image: ${DOCKER_IMG}
+    entrypoint:
+    - tail
```

Comment:
Why is this entrypoint `tail`?

Reply:
This is just a dummy process to keep the container running so that we can exec commands in the container. I'm open to suggestions on that though.